### PR TITLE
add changes :ship:

### PR DESCRIPTION
**Pull request by azat-co**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,14 +12,13 @@ Installing Node and npm
...
 
 ## Instructions
 
 1. Install Node v5.1.0 with one of the methods listed below
 2. Install npm v2.14.15 (typically comes with Node so you rarely if ever need to install it separately)
-3. Check versions 5.1.0 and 2.14.15 for Node and npm respectively 
-
+3. Check versions 5.1.0 and 2.14.15 for Node and npm respectively
 
 ### One-Click Installers
```
Comment (victhevenot) [inline README.md:31]: Is there a way to update Node if you already have it installed? You covered how to update NPM but what about node? do I just need to user the one-click installer again?

Comment (victhevenot) [inline README.md:31]: we discourage students from using MacPorts (and post won't even know what it is) - you should be able to delete this section

Comment (victhevenot) [inline README.md:31]: I would add context for why you want to install with a tar file, from a github repo, why want multiple versions of node.

Comment (victhevenot) [inline README.md:31]: compare NAVE to RVM (ruby version manager) to students understand what the purpose is for

Comment (victhevenot) [inline README.md:31]: You've listed so many ways to install Node, which is great. Make sure you voice your opinion about which way is the best and why.

**Comment by azat-co** [comment]
CC @AnnJohn @vicfriedman updated readme according to feedback and changed tests from tape to mocha

**Comment by victhevenot** [comment]
Hey @azat-co everything look great here. The only comment here is that I would remove the tests. Students don't know how to read the tests yet, so until they hit that point, it's good to remove those. Maybe just tell students to enter `npm` and `node -v` to see if things  were installed

**Pull request closed without merging**